app/javascript/components/Payment/Payment.js

```diff
@@ -100,8 +102,12 @@ export class Payment extends Component {
   // user follows external link like email
   setDefaultPaymentType = () => {
     const urlInfo = window.champaign.personalization.urlParams;
+    // TODO: get the country name from store.
+    const country = document.getElementsByName('country')[0].value;
```

Comment:
The `Payment` component is connected to the store, so we should be getting the country value in `mapStateToProps`.

Reply:
@shivashankar-f22labs can you follow up on @eyko's comment?

Reply:
Yes. I have done the changes. Will raise new PR